frontend/src/metabase/visualizations/click-actions/Mode/Mode.ts

```diff
@@ -43,4 +47,14 @@ export class Mode {
       return actions;
     }
   }
+
+  isDrillEnabled = (drill: DrillThruDisplayInfo): boolean => {
+    const mode = this._queryMode;
+
+    if (mode.hasDrills && Array.isArray(mode.hasDrills)) {
```

Comment:
```suggestion
    if (Array.isArray(mode.hasDrills)) {
```

Does this work? It looks more concise this way.